### PR TITLE
Add HOP to Base

### PR DESCRIPTION
**Pull request by shanefontaine**
Adding the address for HOP on Base ([Etherscan link](https://basescan.org/address/0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC)).

### CODE REVIEW

**Comment by shanefontaine** [comment]
cc @roberto-bayardo

File: data/HOP/data.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
-  "name": "HOP",
+  "name": "Hop",
   "symbol": "HOP",
   "decimals": 18,
-  "description": "A governance token",
+  "description": "A multi-chain token that powers the Hop ecosystem",
+
   "website": "https://hop.exchange",
   "nobridge": true,
```
Comment (roberto-bayardo) [inline data/HOP/data.json:8]: this token seems bridgable... can you remove this line and provide the appropriate L2 bridge address within the base subsection? (and I guess optimism too assuming it's bridgable to optimism?)

Comment (roberto-bayardo) [reply data/HOP/data.json:8]: Oh hmm, I guess your bridge may not compatible with the OptimismERC20 standard?

Comment (roberto-bayardo) [reply data/HOP/data.json:8]: If so maybe you can leave it as "nobridge" here but clarify in the description that it's "bridged by Hop network", kind of like some of the wormhole tokens do: https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/71c0a72c83c6129a6c115caa824a1abe11cafb44/data/APT/data.json#L5

Comment (roberto-bayardo) [reply data/HOP/data.json:8]: looks like our replies crossed paths, consulting with the OP labs folks there isn't a great process for non-standard bridged tokens right now, so let's just leave it as-is only please update the description to clarify it's bridged by you all.

Comment (shanefontaine) [reply data/HOP/data.json:7]: @roberto-bayardo sounds good. Updated the description and will wait for further instruction. Let me know if you need anything 👍

**Comment by roberto-bayardo** [comment]
Apologies for the holdup here. We still don't have a policy around how or if to approve Base tokens that are not bridged by the standard bridge and are in the middle of figuring it out.

**Comment by roberto-bayardo** [comment]
Hi @shanefontaine so that we might understand the bridging process here, can you share txids of a successful bridging flow of HOP between mainnet and Base? Thanks.

**Comment by shanefontaine** [comment]
@roberto-bayardo no problem. Here are some reference txs:

* L1 to Base tx
  * [Source tx](https://etherscan.io/tx/0xfe846a4587f945f39b84c2dbc3acbb69ac0a82898e7d220226dd785ba234eace)
  * [Dest tx](https://basescan.org/tx/0x589ae17e7cf9937d9fc174a343e3ae2f53632c8ee53134910fce4a74e9d02c4f)
* Base to L1 tx
  * [Source tx](https://basescan.org/tx/0x4cfc9945edb78214a180bb6a09e88a0974aec4933fbbf3223afaf78788f1d300)
  * [Dest tx](https://etherscan.io/tx/0x6bdeed83ed52ba1c5663d0403c375fdd41566a8fa2c6ec5dde62aafd76219b23)

The bridge works the same way on Optimism as well. Happy to provide any further info that is needed.

**Comment by ericm-cb** [comment]
Hey @shanefontaine , thanks for all the info! 

Just a few last questions in regards to the [multisig](https://etherscan.io/address/0x1ec078551A5ac8F0f51fAc57Ffc48Ea7d9A86E9d#code) that controls the timelock contract:  

1. How are the private keys stored (cold, hot)? 
2. Are they stored and secured in separate locations? 
3. Are each of the keys held by privileged, trusted individuals and can only be actioned by them?

Thank you!

**Comment by shanefontaine** [comment]
@roberto-bayardo @ericm-cb of course:

1. All the keys are cold wallets used only for Hop-related txs.
2. All of the keys are stored by their owners in separate locations. Each owner has independently implemented their security measures based on years of key management best practices and watchings/learnings as the industry has grown over the years (even learnings from [Coinbase's practices](https://www.wired.com/story/coinbase-physical-vault-to-secure-a-virtual-currency/)),
3. Yes. Each key owner is a trusted team member and the sole handler of the key.

And for some additional info, the Multisig is a v1.1.1 SAFE that has been in production with no issues since Dec 13, 2019 and was both [externally](https://github.com/safe-global/safe-contracts/blob/v1.1.1/docs/audit_1_1_1.md) and internally audited. the Timelock can control some limited parameters of the bridge as I'm sure you've seen, but cannot, for example, drain the bridge of user funds. The Timelock is the industry-standard contract (used by Compound and others) with a 1-day delay that is fully public/onchain and cancellable. This contract has also been audit [externally](https://blog.openzeppelin.com/compound-finance-patch-audit) and internally.

**Review by roberto-bayardo — APPROVED** [review]
Good to go from Base side!

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**